Wabbajack/Settings.cs

```diff
@@ -93,6 +94,18 @@ public class CompilerSettings
         public AbsolutePath OutputLocation { get; set; }
         public MO2CompilationSettings MO2Compilation { get; } = new MO2CompilationSettings();
     }
+    
+    [JsonName("FiltersSettings")]
+    [JsonObject(MemberSerialization.OptOut)]
+    public class FiltersSettings : ViewModel
+    {
+        public bool showNSFW { get; set; }
+        public bool OnlyInstalled { get; set; }
+        public string Game { get; set; }
+        public string Search { get; set; }
+        private bool _isPersistent = true;
+        public bool isPersistent { get => _isPersistent; set => RaiseAndSetIfChanged(ref _isPersistent, value); }
+    }
```

Comment:
`showNSFW` should be `ShowNSFW`, as it's public.  Same with `IsPersistent`

Reply:
Hey nice catch, i am new with github is there any way i can attach a commit to this review or simply commiting again is fine ?

Reply:
If you make another commit with the changes and just push, it'll sync and work out

Reply:
ok thanks !